packages/create-cloudflare/templates/nuxt/templates/wrangler.toml

```diff
@@ -0,0 +1,2 @@
+name = "<TBD>"
+compatibility_date = "<TBD>"
```

Comment:
```suggestion
compatibility_date = "<TBD>"

```

Reply:
Don't we also want to include (commented-out) bindings?

Like this: https://github.com/cloudflare/workers-sdk/blob/7412ce7e6674c37b4263319bb255064c5702321f/packages/create-cloudflare/templates/hello-world/js/wrangler.toml#L5-L51